pkg/sdk/testint/external_volumes_gen_integration_test.go

```diff
@@ -0,0 +1,605 @@
+package testint
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/testenvs"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_ExternalVolumes(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	awsBaseUrl := testenvs.GetOrSkipTest(t, testenvs.AwsExternalBucketUrl)
```

Comment:
Not exactly sure what these evaluate to on your side. The formats I was using locally - 
```
TEST_SF_TF_AWS_EXTERNAL_BUCKET_URL=s3://my_example_bucket
TEST_SF_TF_AWS_EXTERNAL_KEY_ID=1234abcd-12ab-34cd-56ef-1234567890ab
TEST_SF_TF_AWS_EXTERNAL_ROLE_ARN=arn:aws:iam::123456789012:role/myrole
TEST_SF_TF_GCS_EXTERNAL_BUCKET_URL=gcs://my_example_bucket
TEST_SF_TF_AZURE_EXTERNAL_BUCKET_URL=azure://123456789.blob.core.windows.net/my_example_container
TEST_SF_TF_AZURE_EXTERNAL_TENANT_ID=123456789
```